submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/action_specific/logout/template.md

```diff
@@ -0,0 +1,41 @@
+# Cross-Site Request Forgery For Logout Functionality
+
+## Overview of the Vulnerability
+
+Cross-Site Request Forgery (CSRF) occurs when requests to the application are submitted on behalf of an authenticated user without their knowledge via crafted, malicious code which can be in the form of a link the user clicks. The application is unable to distinguish between the malicious request and a legitimate request from the user.
+
+CSRF is possible within this application, allowing an attacker to log-out a valid user. Additionally, the attacker needs to socially engineer the user to click on a link, or paste the malicious code into the user’s browser. If successful, the code will execute within that user’s browser in the context of this domain, logging the user out of their session. An attacker can deny service to users using this CSRF vector to prevent access to the application and constantly logging users out.
+
+## Business Impact
+
+CSRF could lead to reputational damage for the business through the impact to customers’ trust due to not being able to reliably access the application. This could also cause indirect financial impacts to the business.
+
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Use a browser to login to the application at: {{URL}}
+1. Navigate to the following URL and logout: {{URL}}
+1. Use the HTTP interception proxy to intercept the request triggered by the logout function
```

Comment:
Removing this step will speed up the researchers submissions and make it more clear for ASE's to triage.